labcodes/django-react-webpack
# Proposal: vite migration

This PR implements a proposal to change from webpack to vite.
We'll need to change the repo's name, though, since we won't be using webpack anymore.
The proposal changes from `django-react-webpack` to `django-react-boilerplate`.

(This PR is based on two open PRs, #57 and #58 , so be sure to merge those before)

**Link to task:** fix #59 

**How to test:**
- go to this branch
- reinstall all dependencies (`make install_dependencies`)
- run the app and check it's working as intended
- run the jest and cypress tests

**Description of your solution:**
I've removed django-webpack-loader, together with webpack configs, and refactored the frontend app and dependencies to build using vite. All the details are described on my [`Using Vite with Django, the simple way`](https://gist.github.com/lucianoratamero/7fc9737d24229ea9219f0987272896a2) gist, and this will need thorough code review, but it seems pretty straightforward.
